webrtc.html

```diff
@@ -4074,90 +4194,123 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
       <p>Identity assertion validation happens when <code><a href=
       "#dom-peerconnection-setremotedescription">setRemoteDescription</a></code>
       is invoked on <code><a>RTCPeerConnection</a></code>. The process runs
-      asynchronously, meaning that validation of an identity assertion does not
+      asynchronously, meaning that validation of an identity assertion might not
       block the completion of <code>setRemoteDescription</code>.</p>
 
       <p>The identity assertion request process involves the following
-      steps:</p>
+      asynchronous steps:</p>
 
       <ol>
-        <li>The <code>RTCPeerConnection</code> instantiates an IdP proxy as
-        described in <a href="#sec.identity-proxy-communications">Identity
-        Provider Selection section</a> and waits for the IdP to signal that it
-        is ready.
+        <li>
+          <p>Only one identity validation can run at a time for an
+          <code>RTCPeerConnection</code>.  If there is an outstanding identity
+          validation in progress, await its completion.  This can happen because
+          the resolution of <code>setRemoteDescription</code> is not blocked by
+          identity validation unless there is a <a
+          href="#target-peer-identity">target peer identity</a>.
+        </li>
+        <li>
+          <p>The <code>RTCPeerConnection</code> loads the identity assertion
+          from the session description and decodes the base64 value, then parses
+          the resulting JSON.  The <var>idp</var> paramter of the resulting
+          dictionary contains a <var>domain</var> and an optional
+          <var>protocol</var> value that identifies the IdP, as described in
+          [[!RTCWEB-SECURITY-ARCH]].</p>
```

Comment:
You should probably try to match tense here. Either be descriptive or imperative but not both (as in the first and second items on this list)